src/WebJobs.Script.WebHost/Management/GetTriggersResult.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using static Microsoft.Azure.WebJobs.Script.WebHost.Management.FunctionsSyncManager;
+
+namespace Microsoft.Azure.WebJobs.Script.WebHost.Management
+{
+    public class GetTriggersResult
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether the sync operation was successful.
```

Comment:
Remove the "sync" here and below since this result isn't for sync.

Might just say "Gets or sets the error string in the case of a failed operation." and "Gets or sets a value indicating whether the operation was successful."

Reply:
What might even be better is for you to move the Success/Error members into a new base class TriggersOperationResult, and have both SyncTriggersResult and GetTrigersResult inherit from that

Reply:
That will make SyncTriggersResult = TriggersOperationResult i.e. SyncTriggersResult will not have any new members:

```
public class SyncTriggersResult : TriggersOperationResult
{
}
```

@mathewc - Will that be okay?

Reply:
Makes sense. Renamed SyncTriggersResult to TriggersOperationResult.
Updated GetTriggersResult to:

```
public class GetTriggersResult : TriggersOperationResult
{
    /// <summary>
    /// Gets or sets the triggers payload.
    /// </summary>
    public string Content { get; set; }
}